src/platform/forms/tests/save-in-progress/SaveInProgressIntro.unit.spec.jsx

```diff
@@ -505,4 +505,46 @@ describe('Schemaform <SaveInProgressIntro>', () => {
 
     tree.unmount();
   });
+
+  it('should properly hide non-authed start when desired', () => {
+    const user = {
+      profile: {
+        savedForms: [
```

Comment:
I'm not sure how much it matters, but I don't think this accurately reflects a state we can be in. If we're not logged in, we won't have `savedForms`.